pkgs/applications/video/vlc/default.nix

```diff
@@ -22,6 +23,7 @@
 with stdenv.lib;
 
 assert (withQt5 -> qtbase != null && qtsvg != null && qtx11extras != null && wrapQtAppsHook != null);
+assert (skins2Support -> freetype != null);
```

Comment:
```suggestion
```

Reply:
This is a check for the above, if the package doesn't exist fine but only if that flag is disabled.

Reply:
This is not required here and we shouldn't add more occurrences of this.

Reply:
And delete both asserts?

Reply:
Yes